frontend/public/components/routes/_create-route.scss

```
@@ -3,3 +3,11 @@
     margin-bottom: 0;
   }
 }
+
+.co-create-route__weight-input {
+  max-width: 100% !important;
+}
+
+.co-create-route__btn-margin {
```

Per BEM naming, this should name the element instead of describing the style it's adding. Something like

```suggestion
.co-create-route__add-service-btn {
```